packages/shared/src/styles/base.css

```
@@ -487,3 +487,7 @@ details.right-icon {
     border-radius: 14px;
   }
 }
+
+input {
+  -webkit-text-fill-color: var(--theme-label-primary);
+}
```

@binalajudiya this should apply the available class from tailwind 🥳 

After this, I think you can revert the changes from the `profile` file, and see if it works as intended.
```suggestion
input {
  @apply bg-clip-text;

  -webkit-text-fill-color: var(--theme-label-primary);
}

```

---

@sshanzel I applied, tested and it worked  🚀

Reverted `profile` file.